iup/__init__.py

```diff
@@ -76,6 +75,22 @@ def with_columns(self, *args, **kwargs):
         result = super().with_columns(*args, **kwargs)
         return orig_class(result)
 
+    @staticmethod
+    def assert_column_name_all(estimate, column_name):
+        """
+        Verify that all columns have a pattern with a common name
```

Comment:
What if there are grouping factors (e.g. geographic area, demographic group, etc.)? In this case, even a forecast would have columns that are not "date" but also don't contain the "column_name."